src/EA.Weee.Web/Areas/AATFReturn/ViewModels/ReusedOffSiteViewModel.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace EA.Weee.Web.Areas.AatfReturn.ViewModels
+{
+    using System;
+    using System.Collections.Generic;
+    using System.ComponentModel.DataAnnotations;
+    using EA.Weee.Web.ViewModels.Shared;
+
+    public class ReusedOffSiteViewModel : RadioButtonStringCollectionViewModel
+    {
+        public Guid OrganisationId { get; set; }
+
+        public Guid ReturnId { get; set; }
+
+        public Guid AatfId { get; set; }
+
+        [Required(ErrorMessage = "You must tell us whether any of this WEEE has gone directly to another site to be reused")]
```

Comment:
Add simple unit test to ensure the require field attribute is present

Reply:
I think this is covered in ReusedOffSiteViewModelTests.cs?